glusterd2/pmap/registry.go

```diff
@@ -148,9 +170,10 @@ func init() {
 	}
 
 	registry = &pmapRegistry{
-		Ports:  make(map[int]brickSet),
-		bricks: make(map[string]int),
-		conns:  make(map[net.Conn]int),
+		Ports:           make(map[int]brickSet),
+		bricks:          make(map[string]int),
+		conns:           make(map[net.Conn]int),
+		notifyFirewalld: true,
```

Comment:
is `notifyFirewalld` is configurable?

Reply:
No. Not right now. It can be added in future if required.

This is also useful in unit tests of pmap package. It's set to false there. We do not want unit tests to actually configure firewalld.